src/styles/index.scss

```diff
@@ -1,3 +1,149 @@
-body {
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: white;
+$starnum: 1,2,3,4;
+
+html,
+body,
+div,
+span,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
   margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  font-weight: inherit;
+  font-style: inherit;
+  font-size: 100%;
+  font-family: inherit;
+  vertical-align: baseline;
+}
+
+.container {
+  margin: 0;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+}
+
+.card {
```

Comment:
each block should be in separate file, 11'th point in checklist

Reply:
done, see 'v2'